.github/workflows/markdownlint.yml

```diff
@@ -0,0 +1,24 @@
+name: Markdown Lint
+
+on:
+  pull_request:
+    paths:
+      - '**/*.md'
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+    - uses: tj-actions/changed-files@v46
```

Comment:
There's an issue with checking only the changed files. The vale lint check currently works like that and it lints only the changed files, but it reports errors in the unedited parts of the changed files as well. 

A win-win solution, IMO, would be to fix the lint errors in a specific folder, and run the lint check only on that folder, and expand coverage as the lint errors are fixed.

Reply:
Upon doing some research, I could not find a reliable solution to run the linter only on the changed lines of a markdown file. Let's keep this for now and improve the workflow in the future if a workaround is found.